### DISCUSSION TITLE
GitHub Copilot could not connect to server. Extension activation failed: "No access to GitHub Copilot found

### DISCUSSION THREAD

**Discussion by ghost** [community/community#37966] [Copilot Conversations] [answered]
<img width="351" alt="image" src="https://user-images.githubusercontent.com/104374789/199217085-c1168cce-8ebf-4ba1-8e7a-d0d4aa2d89d4.png">

As I am a verified student. I have free access to github copilot.
<img width="636" alt="image" src="https://user-images.githubusercontent.com/104374789/199217251-e6286f1a-8e43-4e42-98e7-3c33726ce7c3.png">

Also, My vscode is updated to version 1.72.2

**Comment by iamcjwilliams** [comment]
Hello, @manastelavane777 If you are still experiencing issues after signing out and signing back in, perform the following (View -> Output; select GitHub Copilot from the pulldown) and provide a screenshot of the error you're receiving

**Reply by ghost** [reply]
I am not getting any output in output section

**Comment by davecheney** [comment, marked as answer]
@manastelavane777 please see https://github.com/community/community/discussions/38029

**Reply by ghost** [reply]
Thanks @davecheney . The second one worked for me.
